demisto_sdk/commands/common/constants.py

```diff
@@ -170,6 +170,9 @@ class FileType(Enum):
 
 CONTENT_FILE_ENDINGS = ['py', 'yml', 'png', 'json', 'md']
 
+IGNORED_FILES = ['__init__.py', 'ApiModules', 'NonSupported', 'Base']  # files to ignore inside Packs folder
```

Comment:
`IGNORED_FILES` - too generic name, ignoring from what?
3/4 are pack names, why `__init__.py` is here? is it also under the `Packs` folder (it shouldn't be)?

Reply:
Fixed